docs/quickstart.rst

```diff
@@ -9,11 +9,9 @@ A Locust performance test is specified in a plain python file:
 .. code-block:: python
 
     import time
-    from locust import HttpUser, task, between
+    from locust import HttpUser, task
 
     class QuickstartUser(HttpUser):
-        wait_time = between(1, 2)
```

Comment:
The removal of a wait time for this example doesn't really make sense (a typical web user - which I think is the most common usecase - doesn't have zero wait time between requests). It can also make a beginner DoS-attack themselves (either their web server, or the Locust client itself if the HTTP-server can keep up) if they simply run the example (case in point #1629).

Reply:
I dont really know what is the issue in that ticket, but it seems he got 100% cpu even when specifying a wait time according to his latest post...

Reply:
True, the #1629 seems to be an unrelated issue, but my point still stands. Even if you don't agree with my statement about it making more sense with a wait time, don't you see an issue with having an example that'll hit 100% CPU and trigger the Locust CPU warning when started?